bin/emqx

```diff
@@ -123,8 +123,8 @@ fi
 # Echo to stderr on errors
 echoerr() { echo "$@" 1>&2; }
 
-# By default, use cuttlefish to generate app.config and vm.args
-CUTTLEFISH="${USE_CUTTLEFISH:-yes}"
+# By default, use hocon to generate app.config and vm.args
```

Comment:
I don't think anyone is actually using hand-crafted app.conf and vm.args any more.
i.e. we can drop this support in 5.0

Reply:
I'll help to remote it.